src/main/kotlin/no/nav/familie/ef/søknad/api/dto/søknadsdialog/EttersendingUtenSøknad.kt

```diff
@@ -0,0 +1,3 @@
+package no.nav.familie.ef.søknad.api.dto.søknadsdialog
+
+data class EttersendingUtenSøknad(val stønadstype: String, val innsending: List<Innsending>)
```

Comment:
Her kan vi gjerne bruke en enum vi har i kontrakter for stønadstype:

```suggestion
data class EttersendingUtenSøknad(val stønadstype: StønadType, val innsending: List<Innsending>)
```